### PR TITLE
Fix: Remove samples replaced by `Toggle between feature request modes` 

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This PR updates `Toggle between feature request modes` in `Features` category, to add redirect URLs and remove remaining screenshots and READMEs.
See PR #1095

## Linked Issue(s)

`common-samples/issues/3304`

## How To Test

Doc only change.

## Links

Below are removed samples - they still exist on the developer website because we didn't remove their READMEs at the time of #1095 .

- https://developers.arcgis.com/ios/swift/sample-code/service-feature-table-cache/
- https://developers.arcgis.com/ios/swift/sample-code/service-feature-table-manual-cache/
- https://developers.arcgis.com/ios/swift/sample-code/service-feature-table-no-cache/

### CODE REVIEW

**Comment by yo1995** [comment]
It might be a good idea to have one of the doc team member to give an approval, to ensure the `redirect_from` pattern is valid.

@TADraeseke do you happen know who spotted this issue in the first place?

**Comment by TADraeseke** [comment]
I'd recommend asking the relevant iOS doc team member to check. Is that @sbaskaran for iOS?

**Review by vquach2404 — APPROVED** [review]

**Comment by sbaskaran** [comment]
@TADraeseke @yo1995 I am not familiar with this, so I checked with Thad. He hasn't done this either, though it looks right.  (Can you also add him to this repo?)

I think we can test it locally this time. We can possibly just update our local config to pull this branch for samples

**Comment by yo1995** [comment]
IMO if @ThadT gives us an OK here, we can proceed. I agree it's tricky to test this out.

As this is the first time we do redirects since the evolution of Citra (new dev website), we shall be meticulous this time to set the tone for future occurrences of redirect links.

**Comment by ThadT** [comment]
I just tried testing this locally. I can see the 'toggle between feature request modes' sample page but the other three links return 404. It might be that I'm incorrect to assume we can test the redirects this way ... it works for guide doc links, but the samples might work differently. Let me see if John can offer some advice.

**Comment by ThadT** [comment]
I created a preview branch to test the redirects.
The only change I made in this branch was to edit the config.yml file to bring in the samples doc from this branch: https://github.com/ArcGIS/afd-native-apis-doc/blob/sample-redirect-preview/documentation/ios/config.yml#L25

In the preview, the redirects aren't working as expected. In my local build, I got 404 errors, but in this case, it just loads the requested sample page (?).
https://sample-redirect-preview.sites.afd.arcgis.com/ios/swift/sample-code/service-feature-table-no-cache/

**Comment by yo1995** [comment]
Thanks Thad for testing. I'm not sure what might be the problem. Maybe we need to bring in John or Pat's help. 

In 2020 I used to be able to find Gatsby's source code, and at that time I saw the redirect function is just concatenating the site's base URL and the string in the `redirect_from` field of metadata json. But now I lost track of in which repo the src is hosted.

**Comment by ThadT** [comment]
@yo1995 - I think you'll find the source code for the site here: https://github.com/ArcGIS/afd-core/tree/master/site/src. I think the redirects are handled by a gatsby plugin. Here's a PR for implementing redirects: https://github.com/ArcGIS/afd-core/pull/1157

We've put this on @jf990 radar and he's been looking into it.

File: arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json [hunk 1]
```diff
@@ -17,7 +17,11 @@
         "AGSServiceFeatureTable.featureRequestMode",
         "AGSServiceFeatureTable.populateFromService(with:clearCache:outFields:completion:)"
     ],
-    "redirect_from": [],
+    "redirect_from": [
```
Comment (jf990) [inline arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]:
```suggestion
    "redirectFrom": [
```

Comment (jf990) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: in the migration to Gatsby we changed this to `redirectFrom` see https://github.com/ArcGIS/afd-core/pull/1157 - - but we need to merge this suggestion and build a -preview and test it first @ThadT

**Review by jf990 — CHANGES_REQUESTED** [review]

File: arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json (hunk 1 above)
Comment (jf990) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: Just spoke with the dev team on this my suggestion won't change anything. Creating a separate issue to address this in https://github.com/ArcGIS/afd-native-apis-doc/issues

Comment (jf990) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: https://github.com/ArcGIS/afd-native-apis-doc/issues/1890

Comment (ThadT) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: Thanks, @jf990

Comment (yo1995) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: With the recent merge of https://github.com/ArcGIS/afd-native-apis-doc/pull/1898 , is it OK to proceed with this PR?

Comment (jf990) [reply arcgis-ios-sdk-samples/Features/Toggle between feature request modes/README.metadata.json:20]: i think we should move forward with it

**Review by sbaskaran — APPROVED** [review]

**Pull request merged**